mpc_webhooks/models/utils.py

```diff
@@ -16,10 +16,44 @@ def _post(url, payload):
         _logger.info("Webhook Timeout[{0}]".format(url))
 
 
+def publish_change(id,model,trigger,dbname=None):
+    try:
+        env_type = None
+        exchange = ""
+        if "staging" in dbname:
+            env_type = "staging"
+        else:
+            env_type = "production"
+
+        topic_name = "odoo_{model}_{env}".format(model=model,env=env_type)
+        _logger.info("topic={topic}".format(topic=topic_name))
+        payload = {
+            'id': id,
+            'trigger': trigger
+        }
+
+        _logger.info("connecting to rabbit")
+        params = pika.URLParameters('amqp://uaoqejpl:vrjdmxg7QDzqcPyS7qAIW5HSme4OEa1s@skunk.rmq.cloudamqp.com/uaoqejpl')
```

Comment:
I have been using the System Parameters in Odoo since discovering that its used mostly for env var configs. 

Parameters can be added by going to `Settings > Technical > System Parameters`

It can be accessed by in code by calling 
`config_env = self.env["ir.config_parameter"].sudo()`
`parameter = config_env.get_param("some.env.parameter", "").strip()`

Reply:
Is there a way at getting that parameter without using self? For now i've left it hardcoded until i can refactor the calls to send_webhook & publish_change